### PR TITLE
Update Console Args doc #1559

### PR DESCRIPTION
**Pull request by kevinsalimi**
It is my first contribution to an open-source project. I want to thanks @adamsitnik for his public speaking in which he presented the open-source world subtly and encouraged me to be a part of the community. Several years down the road, I will come back here to watch this moment.

### CODE REVIEW

**Comment by dnfadmin** [comment]
[CLA assistant check](https://cla.dotnetfoundation.org/dotnet/BenchmarkDotNet?pullRequest=1645) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.dotnetfoundation.org/dotnet/BenchmarkDotNet?pullRequest=1645) before we can accept your contribution.<br/><br/>:x: kevinsalimi [sign now](https://cla.dotnetfoundation.org/dotnet/BenchmarkDotNet?pullRequest=1645)<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.dotnetfoundation.org/check/dotnet/BenchmarkDotNet?pullRequest=1645) it.</sub>

File: docs/articles/guides/console-args.md [hunk 1]
```diff
@@ -101,6 +101,7 @@ You can also filter the benchmarks by categories:
 ## Diagnosers
 
 * `-m`, `--memory` - enables MemoryDiagnoser and prints memory statistics
+* `-t`, `--threading` - enables MemoryDiagnoser and Prints threading statistics
```
Comment (adamsitnik) [inline docs/articles/guides/console-args.md:104]: `MemoryDiagnoser` -> `ThreadingDiagnoser` 

```suggestion
* `-t`, `--threading` - enables `ThreadingDiagnoser` and prints threading statistics
```

File: docs/articles/guides/console-args.md [hunk 2]
```diff
@@ -193,22 +194,34 @@ dotnet run -c Release -- --filter * --runtimes netcoreapp2.0 netcoreapp2.1 --sta
...
-* `-a`, `--artifacts` Valid path to accessible directory
-* `--outliers` (Default: RemoveUpper) DontRemove/RemoveUpper/RemoveLower/RemoveAll
-* `--affinity` Affinity mask to set for the benchmark process
-* `--allStats` (Default: false) Displays all statistics (min, max & more)
-* `--attribute` Run all methods with given attribute (applied to class or method)
-* `--monoPath` custom Path for Mono
-* `--cliPath` custom Path for dotnet cli
+* `-j`, `--job` (Default: Default) Dry/Short/Medium/Long or Default.
+* `-e`, `--exporters` GitHub/StackOverflow/RPlot/CSV/JSON/HTML/XML.
+* `-i`, `--inProcess` (default: false) run benchmarks in Process.
```
Comment (adamsitnik) [inline docs/articles/guides/console-args.md:199]: While reading the old description now, I think that we should improve it:

```suggestion
* `-i`, `--inProcess` (default: false) run benchmarks in the same process, without spawning child process per benchmark.
```

File: docs/articles/guides/console-args.md [hunk 3]
```diff
@@ -193,22 +194,34 @@ dotnet run -c Release -- --filter * --runtimes netcoreapp2.0 netcoreapp2.1 --sta
...
+* `-a`, `--artifacts` valid path to accessible directory.
```
Comment (adamsitnik) [inline docs/articles/guides/console-args.md:200]:
```suggestion
* `-a`, `--artifacts` valid path to an accessible directory where output artifacts will be stored.
```

File: docs/articles/guides/console-args.md [hunk 4]
```diff
@@ -193,22 +194,34 @@ dotnet run -c Release -- --filter * --runtimes netcoreapp2.0 netcoreapp2.1 --sta
...
-* `--help` Display this help screen.
-* `--version` Display version information.
-* `--keepFiles` Determines if all auto-generated files should be kept or removed after running the benchmarks.
-* `--noOverwrite` Determines if the exported result files should not be overwritten.
-* `--disableLogFile` Disables the logfile.
-* `--maxWidth` Max paramter column width, the default is 20.
+* `--stopOnFirstError` stop on first error.
+* `--help` display this help screen.
+* `--version` display version information.
+* `--keepFiles` determines if all auto-generated files should be kept or removed after running the benchmarks.
```
Comment (adamsitnik) [inline docs/articles/guides/console-args.md:215]:
```suggestion
* `--keepFiles` (default: false) determines if all auto-generated files should be kept or removed after running the benchmarks.
```

File: docs/articles/guides/console-args.md [hunk 5]
```diff
@@ -193,22 +194,34 @@ dotnet run -c Release -- --filter * --runtimes netcoreapp2.0 netcoreapp2.1 --sta
...
+* `--noOverwrite` determines if the exported result files should not be overwritten.
```
Comment (adamsitnik) [inline docs/articles/guides/console-args.md:216]:
```suggestion
* `--noOverwrite` (default: false) determines if the exported result files should not be overwritten.
```

**Review by adamsitnik — APPROVED** [review]
LGTM, big thanks for improving our docs @kevinsalimi !

**Review by adamsitnik — APPROVED** [review]

**Pull request merged**